### ISSUE TITLE
plan action fails when plan is too long

### ISSUE THREAD

**Issue by mentos1386** [dflook/terraform-github-actions#133]
I'm trying to run a plan for a bunch of changes, which results in successful plan, but github action fails due to plan being too big.

```
 ##[debug]/tmp/github_pr_comment.stderr::memo: Plan generated in [***](https://github.com/***/***/actions/runs/***)
##[debug]/tmp/github_pr_comment.stderr:{"message":"Validation Failed","errors":[{"resource":"IssueComment","code":"custom","field":"body","message":"body is too long (maximum is 65536 characters)"}],"documentation_url":"https://docs.github.com/rest/reference/issues#update-an-issue-comment"}
##[debug]/tmp/github_pr_comment.stderr:Traceback (most recent call last):
##[debug]/tmp/github_pr_comment.stderr:  File "/usr/local/bin/github_pr_comment", line 490, in <module>
##[debug]/tmp/github_pr_comment.stderr:    main()
##[debug]/tmp/github_pr_comment.stderr:  File "/usr/local/bin/github_pr_comment", line 471, in main
##[debug]/tmp/github_pr_comment.stderr:    comment_url = update_comment(issue_url, comment_url, body, only_if_exists)
##[debug]/tmp/github_pr_comment.stderr:  File "/usr/local/bin/github_pr_comment", line 353, in update_comment
##[debug]/tmp/github_pr_comment.stderr:    response.raise_for_status()
##[debug]/tmp/github_pr_comment.stderr:  File "/usr/lib/python3/dist-packages/requests/models.py", line 943, in raise_for_status
##[debug]/tmp/github_pr_comment.stderr:    raise HTTPError(http_error_msg, response=self)
##[debug]/tmp/github_pr_comment.stderr:requests.exceptions.HTTPError: 422 Client Error: Unprocessable Entity for url: https://api.github.com/repos/***/***/issues/comments/977784576
```

No idea what would be the desired outcome. Truncated comment message with a link to run to check the full plan?

**Comment by dflook** [comment]
Thanks @mentos1386, I had this problem in the back of my mind but wasn't sure anyone would hit it 😄.
I think truncating it and referring to the the full plan in the workflow log is good idea.

**Comment by mentos1386** [comment]
One thing that also comes to my mind is, to store plan as an artifact, and link to the artifact. Is this doable even? It might be a bit nicer UX then to link to the action execution, but not that much nicer.

**Comment by tenpaiyomi** [comment]
I came across this issue as well.  The solution I wound up doing in my setup was to grab the comments for the applicable PR, find all of those which matched a filter (in this case, I found all that has some text within the body and was created by the github actions bot user), deleted those, and then created a new comment with the status of changes pending and a link to the specific job id, such that the user could click the link and it would take them directly to the action that contained the terraform plan step, where they could view it from there.

Not ideal, but worked to get around the too long comment issue.
